test/demo/model.test.js

```diff
@@ -307,27 +311,36 @@ describe.skip('Game Model', () => {
       });
     });
 
-    describe('for invalid words', () => {
+    describe("for invalid words", () => {
       const getWord = (model) => {
         const letter = model.letterBank[0];
-        return letter.repeat(model.letterBank.filter((l) => {
-          return l === letter;
-        }).length + 1);
+        return letter.repeat(
+          model.letterBank.filter((l) => {
+            return l === letter;
+          }).length + 1
+        );
       };
 
-      it('it returns null', () => {
+      it("it returns null", () => {
         const model = getModel();
         const word = getWord(model);
-
+<<<<<<< HEAD
+        console.log(word);
         expect(model.playWord(word)).toBe(null);
-        expect(model.playWord('123')).toBe(null);
+        expect(model.playWord("123")).toBe(null);
+        expect(model.playWord("")).toBe(null);
+=======
```

Comment:
Think this was leftover from a merge conflict. Make sure to completely remove any of the conflict syntax from your PR!